content/industry/index.md

```diff
@@ -18,6 +18,14 @@ The Other Use section below describes a few smaller examples of TLA⁺ use that
 
 Examples can be added to this page by [opening a pull request on GitHub](https://github.com/tlaplus/foundation/pulls) or emailing foundation@tlapl.us.
 
+## Google
+
+Google engineers used TLA+ and the TLAPS proof system to formally verify the _Jayanti–Tarjan Concurrent Union-Find_ data structures, which are deployed for large-scale clustering over tens of billions of data points.
```

Comment:
Use unicode ⁺ instead of + for TLA⁺, matching rest of document.

Reply:
On Linux at least you can type this with ctrl+shift+u then typing the unicode codepoint code, which in this case is 207a (I have ended up just remembering this).